plugins/org.eclipse.n4js/src/org/eclipse/n4js/external/TypeDefinitionGitLocationProvider.java

```diff
@@ -101,7 +100,8 @@ public boolean equals(final Object obj) {
 		 * The default location for the type definition file. Used in the production code.
 		 */
 		// TODO: Do NOT forget to change this before merging!
-		PUBLIC_DEFINITION_LOCATION("n4jsd", "https://github.com/NumberFour/n4jsd.git", GitUtils.getMasterBranch()),
+		// PUBLIC_DEFINITION_LOCATION("n4jsd", "https://github.com/NumberFour/n4jsd.git", GitUtils.getMasterBranch()),
+		PUBLIC_DEFINITION_LOCATION("n4jsd", "https://github.com/qtran-n4/n4jsd.git", "IDE-2850"),
```

Comment:
`TODO: Do NOT forget to change this before merging!` - I wonder what it means ;)